README.rst

```diff
@@ -31,42 +31,61 @@
 
 |
 
-PyScaffold helps you setup a new Python project. Checkout out `this demo project`_, which was set up using Pyscaffold.
-In order to install PyScaffold, just pick your favourite installation method::
+PyScaffold is the tool of choice for bootstrapping high quality Python
+packages, ready to be shared on PyPI_ and installed via pip_.
+It is easy to use and encourages the adoption of the best tools and
+practices of the Python ecosystem, helping you (and your team, if your have one)
+to stay sane, happy and productive. The best part? It is stable and been used
+in the wild for over half a decade!
```

Comment:
This paragraph tries to summarise *what PyScaffold is* (a tool for bootstraping Python projects), who is its *target audience* (developers and teams that want to stay sane, happy and productive), and *what it can do for them* (help to build high quality packages that are ready to be shared over PyPI and installed with pip and that adopt the best practices). I also could not resist to reassure everyone that PyScaffold is not some sort of new kid in the block...

Reply:
Sounds good. What about `helping you and your team` instead of the parentheses? It simplifies it and even if the reader has no team, I think it's not much of a problem. 
`and been used in the wild` -> `and has been used by millions of developers over the globe for over ...`.

Reply:
😄 that is a bold one! How about thousands? At least we have the download counter to back the claim hahaha

Reply:
okay, let's say thousands. The overall downloads number is at more than 10 million, so I thought millions of developers is fine but yeah, maybe thousands is more correct.